src/CoreBundle/Resources/public/javascript/tableEditor.js

```diff
@@ -812,7 +808,19 @@ function SetChangedDataMessage() {
     CHAMELEON.CORE.MTTableEditor.initInputChangeObservation();
 }
 
+CHAMELEON.CORE.MTTableEditor.initHelpTexts = function () {
+    $(".help-text-button").click(function () {
+        var helpTextId = '#tooltip-' + $(this).attr("data-helptextId");
```

Comment:
The term `tooltip` is used multiple times, but I think this is too generic (and the help text isn't even a tooltip). Can we replace `tooltip` with `helptext` or `help-text`?

Reply:
changed in helptext
I have also added this functionality to fields across the entire width.